src/App.js

```diff
@@ -1,25 +1,88 @@
 import logo from './logo.svg';
 import './App.css';
+import IdCard from './components/IdCard';
+import Greetings from './components/Greetings';
+import Random from './components/Random';
+import BoxColor from './components/BoxColor';
+import CreditCard from './components/CreditCard';
+import Rating from './components/Rating';
+import DriverCard from './components/DriverCard';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+  <IdCard
+  lastName='Doe'
+  firstName='John'
+  gender='male'
+  height={1.78}
+  birth={new Date("1992-07-14")}
+  picture="https://randomuser.me/api/portraits/men/44.jpg"
+/>
+
+<IdCard
+  lastName='Delores '
+  firstName='Obrien'
+  gender='female'
+  height={1.72}
+  birth={new Date("1988-05-11")}
+  picture="https://randomuser.me/api/portraits/women/44.jpg"
+/>
+
+<Greetings lang="de">Ludwig</Greetings>
+<Greetings lang="fr">François</Greetings>
+ 
+<Random min={1} max={6}/>
+<Random min={1} max={100}/>
+
+<BoxColor r={255} g={0} b={0} />
+<BoxColor r={128} g={255} b={0} />
+
+<CreditCard
+  type="Visa"
+  number="0123456789018845"
+  expirationMonth={3}
+  expirationYear={2021}
+  bank="BNP"
+  owner="Maxence Bouret"
+  bgColor="#11aa99"
+  color="white" 
+/>
```

Comment:
forgot to the pass the other 2 credit card that they give you on the lab instructions